src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -426,8 +445,35 @@ public Command getDriveToPoseCommand() {
     return AutoBuilder.pathfindToPose(targetPose, constraints, 0.0);
   }
 
+  public void rotationallyLockedIn() {
```

Comment:
Could we rename this to `alignToTarget` or similar? As written, it is ambiguous as to whether or not it will return a boolean